Makefile

```diff
@@ -102,9 +102,14 @@ tag-prod:
 
 # OpenShift Aliases
 
-server-prep: # TODO: Move role binding command to a server prep template
+add-role:
+	@oc policy add-role-to-user admin system:serviceaccount:$(TARGET_NAMESPACE):default -n $(TOOLS_NAMESPACE)
```

Comment:
As a newcomer should I know what these commands are used for? Perhaps there's some room for documentation here?

Reply:
Lots of room for documentation. We were hoping for these to be roughly self-explanatory but failed on that front. Many of these were added as we found gaps between our Makefile targets and what was required to fully stand up our infra.